src/test/java/com/artipie/rpm/RpmTest.java

```diff
@@ -28,22 +28,18 @@
 import com.artipie.asto.Key;
 import com.artipie.asto.Storage;
 import com.artipie.asto.memory.InMemoryStorage;
+import com.jcabi.xml.XMLDocument;
 import java.nio.charset.Charset;
-import java.nio.file.Path;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.core.IsEqual;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.io.TempDir;
 
 /**
  * Unit tests for {@link Rpm}.
  *
  * @since 0.9
- * @todo #190:30min Don't change metadata when invalid package is sent.
- *  Currently Rpm is recalculating metadata when an invalid package is sent.
- *  It should not. Correct that and enable the test below.
- *  To do that `InvalidPackageException` should be caught when parsing packages.
  */
 final class RpmTest {
```

Comment:
@olegmoz also we have to make sure that incremental update behaves the same: please, add todo to add corresponding test method and fix.